backend/game/scribble/handler/gamemaster/gamemaster.go

```diff
@@ -364,6 +365,7 @@ type RequestCurrentGameInfoSend struct {
 	HintString     string        `json:"hintString"`
 	MaxRounds      int           `json:"maxRounds"`
 	TimerRemaining time.Duration `json:"timerRemaining"`
+	selectedClient client        `json:"selectedClient"`
```

Comment:
I've added this 'selectedClient' to be part of the struct that gets sent to you now @gordon-cheung